front/src/modules/trainschedule/components/Timetable/TimetableTrainCard.tsx

```diff
@@ -150,173 +138,131 @@ const TimetableTrainCard = ({
-            >
-              {train.mechanicalEnergyConsumed}&nbsp;kWh
-            </small>
-
-            <span className="mr-xl-3 mr-lg-2">{train.pathLength}</span>
+        </div>
 
-            <div className="text-nowrap text-right">
-              <MdAvTimer />
-              <span className="ml-1" data-testid="train-duration">
-                {sec2time(train.duration / 1000)}
+        {train.isValid && (
+          <div className="scenario-timetable-train-body">
+            <div className="more-info">
+              <span className="mr-4">{t('timetable.stopsCount', { count: train.stopsCount })}</span>
+              {train.pathLength && <span className="mr-4">{train.pathLength}</span>}
+              <span className="mr-4" data-testid="allowance-energy-consumed">
+                {train.mechanicalEnergyConsumed}&nbsp;kWh
               </span>
             </div>
-          </div>
-          {train.labels.length > 0 && (
-            <div className="scenario-timetable-train-tags">
-              {train.labels.map((tag) => (
-                <div className="scenario-timetable-train-tags-tag" key={nextId()}>
-                  {tag}
-                </div>
-              ))}
+            <div className="duration-time">
+              <span data-testid="train-duration">
+                {dayjs.duration(train.duration).format('HH[h]mm')}
+              </span>
             </div>
-          )}
-        </div>
-
-        <div className="scenario-timetable-train-buttons">
-          <button
-            className="scenario-timetable-train-buttons-selectprojection"
-            type="button"
-            aria-label={t('timetable.choosePath')}
-            title={t('timetable.choosePath')}
-            onClick={selectPathProjection}
-          >
-            <GiPathDistance />
-          </button>
-          <button
-            className="scenario-timetable-train-buttons-duplicate"
-            type="button"
-            aria-label={t('timetable.duplicate')}
-            title={t('timetable.duplicate')}
-            onClick={duplicateTrain}
-          >
-            <MdContentCopy />
-          </button>
-          <button
-            className="scenario-timetable-train-buttons-update"
-            type="button"
-            aria-label={t('timetable.update')}
-            title={t('timetable.update')}
-            onClick={editTrainSchedule}
-            data-testid="edit-train"
-          >
-            <Pencil />
-          </button>
-          <button
-            className="scenario-timetable-train-buttons-delete"
-            type="button"
-            aria-label={t('timetable.delete')}
-            title={t('timetable.delete')}
-            onClick={deleteTrain}
-          >
-            <Trash />
-          </button>
-        </div>
+          </div>
+        )}
+      </div>
+      <div className="scenario-timetable-train-buttons">
+        <button
+          className="scenario-timetable-train-button-1"
```

Comment:
You can select the first child using the css pseudo class `first-child` instead of attach a specif class. You can see the suggestion below ;)
```suggestion
          className="scenario-timetable-train-button"
```

Reply:
Thanks for the tip, it's done